### ISSUE TITLE
summary data without EAF

### ISSUE THREAD

**Issue by iamyingzhou** [jwr-git/pwcoco#4]
Excuse me, I have a question regarding the use of pwcoco. My summary data does not have the EAF column, and when I tried assigning NA to the EAF column, the program threw an error. 

I am also wondering if it's possible to use the MAF column as the EAF column, since sometimes my data only has the MAF column.

Thank you very much for your time and expertise in helping me with this issue.

**Comment by jwr-git** [comment]
Hi, yes it should be possible to use any allele frequency with PWCoCo, so long as the frequency corresponds to the first allele in the file. I will change the documentation to reflect this.
Thanks!

**Comment by iamyingzhou** [comment]
Could you kindly confirm whether the frequency  column is required for PwCoco to function properly? Would it be feasible to run PwCoco without this column?  thanks a lot!

**Comment by jwr-git** [comment]
Allele frequencies are required for PWCoCo to function for the following reasons:
1) To estimate the variance of the phenotype
2) To ensure alignment/harmonisation with the reference panel
3) MAF filtering (if required)

In the future, I plan to remove the requirement for the allele frequencies to be provided by the user. However, in the meantime, if you do not have frequencies for your data, you can simply use your reference panel and Plink to infer the allele frequencies and provide these for PWCoCo instead.

**Comment by iamyingzhou** [comment]
Thank you for your prompt response! I observed that in the coloc package, the MAF information is not mandatory when providing case-control summary data. It is sufficient to only provide the BETA, SE, and PVAL values. Furthermore, EAF and MAF may not be considered crucial in the coloc package, as the algorithm is based on the computation of FREQUENCY*(1-FREQUENCY). I would like to kindly inquire if PwCoco is based on the coloc package or if substantial modifications have been made to its implementation?

**Comment by jwr-git** [comment]
No, there are not any changes to the implemention of the colocalisation analysis. In my comment above, I have told you which functionalities require allele frequencies.
